src/HFO.hpp

```diff
@@ -63,6 +64,11 @@ struct Config {
   std::vector<int> defense_nums; // Defensive player numbers
 };
 
+struct Player {
+  rcsc::SideID side;    // SideID is an enum: 1 = LEFT, 0 = NEUTRAL, -1 = RIGHT 
```

Comment:
For now I think it's best to introduce our own enum. This way we don't require users to include rcsc libraries just to get a single enum.